src/main/java/org/mockbukkit/mockbukkit/plugin/PluginManagerMock.java

```diff
@@ -564,7 +583,7 @@ private MockBukkitURLClassLoader createURLClassLoader(File pluginFile, PluginDes
 			}
 		}
 		throw new FileNotFoundException(
-				"Could not find file plugin.yml. Maybe forgot to add the 'main' property?");
+				"Could not find file plugin.yml or paper-plugin.yml. Maybe forgot to add the 'main' property?");
```

Comment:
Minor suggestion for consistency:
 ```suggestion
				"Could not find file paper-plugin.yml or plugin.yml. Maybe forgot to add the 'main' property?");
```